src/main/resources/modules/total_joint_replacement/functional_status_assessments.json

```diff
@@ -0,0 +1,1141 @@
+            "condition_type": "Attribute",
+            "attribute": "assessment_done",
+            "operator": "==",
+            "value": "PROMIS-29"
+          }
+        },
+        {
+          "transition": "VR12_Mental_Health_2",
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "assessment_done",
+            "operator": "==",
+            "value": "VR-12"
+          }
+        },
+        {
+          "transition": "VR36_Mental_Health",
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "assessment_done",
+            "operator": "==",
+            "value": "VR-36"
+          }
+        },
+        {
+          "transition": "First_FSA_Hip"
+        }
+      ]
+    },
+    "VR12_Total_Assessment": {
+      "type": "MultiObservation",
+      "category": "survey",
+      "number_of_observations": 8,
+      "codes": [
+        {
+          "system": "LOINC",
+          "code": "71934-4",
+          "display": "Veterans Rand health survey - 12 item (VR-12)"
+        }
+      ],
+      "direct_transition": "VR12_Attribute_Set",
+      "remarks": [
+        "Eight different subscales are scored separately: Mental Health, Role Emotion, Social Functioning, Vitality, General Health, Bodily Pain, Role Physical, and Physical Functioning. Each have a score between 4 - 20."
+      ]
+    },
+    "KOOS_Attribute_Set": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "KOOS"
+    },
+    "PROMIS10_Attribute_Set": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "PROMIS-10"
+    },
+    "PROMIS10_Attribute_Set_2": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "PROMIS-10"
+    },
+    "PROMIS29_Attribute_Set": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "PROMIS-29"
+    },
+    "VR12_Attribute_Set": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "VR-12"
+    },
+    "VR12_Attribute_Set_2": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "VR-12"
+    },
+    "VR36_Attribute_Set": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "VR-36"
+    },
+    "HOOS_Attribute_Set": {
+      "type": "SetAttribute",
+      "attribute": "assessment_done",
+      "direct_transition": "FSA_Performed",
+      "value": "HOOS"
+    },
+    "FSA_Performed": {
+      "type": "Simple",
+      "direct_transition": "Joint_Replacement_Assessment_Terminal"
+    },
+    "PROMIS10_GPH": {
```

Comment:
Is there any difference between the knee and hip version of the PROMIS10 and VR12 mental health tracks? If not it seems like you could save a lot of duplication by having both hip and knee transition to the same state.

Reply:
They are the same. Thanks, I'll fix it.